src/schema/posts.ts

```diff
@@ -1028,23 +1028,23 @@ export const resolvers: IResolvers<any, Context> = {
 
         if (content.length > MAX_CONTENT_LENGTH) {
           throw new ValidationError(
-            'Content has a maximum length of 80 characters',
+            'Content has a maximum length of 4000 characters',
```

Comment:
Title also has a max length perhaps we should also check that here?

Reply:
There is, we are already checking it from 1024. I just fixed the typo here.